Trustroots/trustroots
# Migrate Tribes page to React (finishing the feature)

#### Proposed Changes

- Finalize the migration of `/tribes` page to React
- api calls are made from React
- there is a remaining layer of angularjs for communicating between angular and react

#### Testing Instructions

- git checkout or switch to `tribes-react-page`, `npm install`, `npm start`
- go to `/tribes`
- See that the page works as expected; that the functionality has not degraded.
  - specifically, test when user is both `signed in` and `signed out`.
- FYI we have not migrated some google analytics calls. So they were removed.

Finishing work on #1133
~Depends on #1137, #1143 and #1144, and whatever these depend on. The dependencies should be merged to `tribes-react` branch first and then this branch should be rebased.~ (update: dependencies merged)
